translations/fr/pack/tsk/tskp.json

```diff
@@ -332,8 +332,8 @@
         "skill_intellect": 5,
         "skill_combat": 2,
         "skill_willpower": 2,
-        "subname": "The Photographer",
-        "text": "You begin the game with Darrell's Kodak in play.\n[fast] During a skill test at your location, spend 1 evidence from an asset you control: Reduce the difficulty of this test by 2. (Limit once per test.)\n[elder_sign] effect: +1. Place 1 evidence on an asset you control.",
+        "subname": "Le Photographe",
+        "text": "Vous commencez la partie avec le Kodak de Darrell en jeu.\n[fast] Lors d'un test de compétence dans votre lieu, dépensez 1 preuve d'un soutien que vous controlez : diminuez de 2 la difficulté de ce test. (Limite d'une fois par test.)\n[elder_sign] effect: +1. Placez 1 preuve sur un soutien que vous contrôlez.",
```

Comment:
> `d'un soutien que vous controlez`
> `Placez 1 preuve sur un soutien que vous contrôlez.`

Different spellings for `contrôlez` ?

Reply:
je vais corriger, bien vu

Reply:
I corrected and added two signature cards of Darrell